Adds indie AI, indi gear pool, list for town locations and 1 location…

MDTorch opened the pull request:
… for indi spawn.

MDTorch committed:
Adds indie AI, indi gear pool, list for town locations and 1 location for indi spawn.

guido2 commented on init.sqf:
```diff
@@ -17,6 +18,9 @@ call compile preprocessFileLineNumbers "tmscti\supply_locations.sqf";
 call compile preprocessFileLineNumbers "tmscti\area_control_installations.sqf";
 
+// Load townlocations
+execVM "tmscti\ai_control\tms_town_locations.sqf";
```
Why execVM instead of call?

guido2 commented on tmscti/ai_control/tms_ai_spawn_and_equip.sqf:
```diff
@@ -0,0 +1,550 @@
+				_nearest_buildings = nearestObjects [_active_location, ["House"], 100];
+				_number_of_buildings = count _nearest_buildings;
+				
+				if (_number_of_buildings < 0) then {
```
Shouldn't this be `if (_number_of_buildings < 0)`?

MDTorch committed:
#94 typo

Changed < to >

guido2 commented on tmscti/ai_control/tms_ai_cycle_waypoints.sqf:
```diff
@@ -0,0 +1,34 @@
+_group = (_this select 0);
```
Please rename these files so they do not have the `tms_ai_` prefix. Since they are inside the directory `tmscti/ai_control/`, this prefix is superfluous.

MDTorch committed:
Fixes everything related to the indie ai defenders

MDTorch committed:
Adds indie ai defenders to every supply location

MDTorch committed:
Update equip_resistance_infantry.sqf

Gives big units of AI defenders a second special weapon and a second heavy weapon

MDTorch committed:
Rename tms_town_locations.sqf to tms_defense_locations.sqf

Better name

MDTorch committed:
Update init.sqf

Changed name of the former "tms_town_locations" to "tms_defense_locations"
